### DISCUSSION TITLE
CamImage ActiveX Control Help

### DISCUSSION THREAD

**Discussion by phauguth** [AxisCommunications/discussions#48] [Axis Windows SDKs] [answered]
Hi, 

I'm trying to integrate video feeds from an Axis M3065-V into a SCADA application through the ActiveX component CamImage. The SCADA program I'm using is Rockwell Factorytalk View SE. When I add this activeX component, there are several parameters that are to be filled out. I've tried just inputting the URL to the URL of the camera directly (http://169.254.56.97/#view in my test). When I do this and try and run the SCADA application it crashes. I'm just wondering if the URL has to be that of the actual stream? Is there a more specific syntax for the URL? 

I've attached an image of the parameters that are available. Do all of these need to be filled out? Or only specific ones? 

Thanks in advance.

![CamImage Activex](https://user-images.githubusercontent.com/112511400/187517948-68fbb359-cc7e-4b96-bd09-04f7e5662a3f.JPG)

**Comment by anders-axis** [comment, marked as answer]
OMG, that is a component that was abandoned over 15 years ago. I'm unsure if there is anyone alive knowing the things you need to know. My best guess is using the MJPEG over HTTP url:
http://myServer/axis-cgi/mjpg/video.cgi

**Reply by phauguth** [reply]
Go figure! The other alternative is using an internet explorer object in my SCADA but when I try that I get "your browser is not supported". Back to the drawing board, I guess.
